instrumentation/spring/spring-web/spring-web-3.1/library/src/main/java/io/opentelemetry/instrumentation/spring/web/v3_1/internal/WebTelemetryUtil.java

```diff
@@ -19,9 +19,23 @@ public class WebTelemetryUtil {
   private WebTelemetryUtil() {}
 
   // allows access to the private field for the spring starter
-  @SuppressWarnings("ConstantField")
+  private static Function<
+          SpringWebTelemetryBuilder,
+          DefaultHttpClientInstrumenterBuilder<HttpRequest, ClientHttpResponse>>
+      getBuilder;
+
   public static Function<
           SpringWebTelemetryBuilder,
           DefaultHttpClientInstrumenterBuilder<HttpRequest, ClientHttpResponse>>
-      GET_BUILDER;
+      getBuilder() {
+    return getBuilder;
+  }
+
+  public static void setGetBuilder(
```

Comment:
`setGetBuilder` is a bit weir what do you think about using `setBuilderExtractor`/`getBuilderExtractor` instead?

Reply:
good idea